libs/ledger-live-common/src/families/bitcoin/wallet-btc/crypto/bitcoin.ts

```diff
@@ -8,6 +8,8 @@ import { InvalidAddress } from "@ledgerhq/errors";
 import { DerivationModes } from "../types";
 import Base from "./base";
 
+const BTC_DUST_LIMIT = 546; // satoshies
```

Comment:
oh so we decided to set a static value and not take txSize into account ?

Reply:
For now yes as it's the value in most wallets
We can improve it in a 2nd PR though